S7.Net/PLC.cs

```diff
@@ -277,10 +419,10 @@ public void ReadMultipleVars(List<DataItem> dataItems)
         {
             int cntBytes = dataItems.Sum(dataItem => VarTypeToByteLength(dataItem.VarType, dataItem.Count));
             
-            if (dataItems.Count > 20)
-                throw new Exception("Too many vars requested");
-            if (cntBytes > 222)
-                throw new Exception("Too many bytes requested"); // TODO: proper TDU check + split in multiple requests
+            //Is this really a limit?
```

Comment:
Did you try with more than 20 dataItems?

Reply:
I did not try but I really can't find this limit anywhere else. Including in libnodave.